src/components/pages/Govern/GovernDAO.tsx

```diff
@@ -0,0 +1,320 @@
+  :hover {
+    outline: none;
+    span {
+      border-bottom: 1px rgb(37, 39, 45) solid;
+    }
+  }
+`;
+
+const StyledToken = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 0.25rem;
+  font-weight: 600;
+  font-size: 1rem;
+  margin-top: 0.5rem;
+
+  span {
+    ${({ theme }) => theme.mixins.numeric};
+  }
+
+  @media (min-width: ${ViewportWidth.s}) {
+    margin-top: 0;
+  }
+`;
+
+const Circle = styled.div<{ color: string }>`
+  width: 1.25rem;
+  height: 1.25rem;
+  border-radius: 0.625rem;
+  margin-right: 0.5rem;
+  background: ${({ color }) => color};
+  overflow: hidden;
+
+  > * {
+    width: 100%;
+    height: 100%;
+  }
+`;
+
+const Address = styled.a`
+  ${({ theme }) => theme.mixins.numeric};
+
+  background: url(${DiagonalArrow}) #f7f8f9 no-repeat;
+  background-position-x: 95%;
+  background-position-y: 50%;
+  background-size: 1.25rem;
+  padding: 0.25rem 2.25rem 0.25rem 0.75rem;
+  border: none;
+  border-radius: 0.75rem;
+
+  &:hover {
+    cursor: pointer;
+
+    span {
+      border-bottom: 1px solid #000;
+    }
+  }
+`;
+
+const NavigateButton = styled.button`
+  background: url(${LeftArrow}) no-repeat;
+  background-position-x: 50%;
+  background-position-y: 50%;
+  width: 1.25rem;
+  height: 1.5rem;
+  margin-right: 0.5rem;
+  border: none;
+  cursor: pointer;
+`;
+
+const StateSelector: FC<{
+  state: State;
+  onClick: (i: State) => void;
+}> = ({ state, onClick }) => {
+  const { title, accent } = mapDataToState(state);
+  return (
+    <StyledRow onClick={() => onClick(state)}>
+      <Circle color={accent ?? '#000'} />
+      <span>{title}</span>
+    </StyledRow>
+  );
+};
+
+const TokenItem: FC<TokenProps> = ({ image, amount, key }) => {
+  const valueRounded = amount && Math.floor(amount.toNumber());
+  return (
+    <StyledToken key={key}>
+      <Circle color="#eee">
+        <img src={image} alt="mta" />
+      </Circle>
+      <CountUp end={valueRounded} decimals={0} />
+    </StyledToken>
+  );
+};
+
+export const GovernDAO: FC = () => {
+  const [state, setState] = useState<State>(DEFAULT);
+
+  const handleNavigateClick = useCallback(() => setState(DEFAULT), []);
+  const handleStateSelect = useCallback((s: State) => setState(s), []);
+
+  const { title, address, tooltip } = mapDataToState(state);
+
+  // mocked
+  const tokenMapping: Map<State, TokenProps[]> = new Map([
+    [MSTABLE, mockTokens],
+    [COMMUNITY, mockTokens],
+    [PROTOCOL, []],
+    [DEVELOPMENT, []],
+  ]);
+  const tokens = tokenMapping.get(state);
+  //
+
+  const items = [MSTABLE, COMMUNITY, PROTOCOL, DEVELOPMENT];
+  const hasTokens = tokens?.length !== 0;
+  const isDefault = state === DEFAULT;
+
+  return (
+    <Container>
+      <Header>
+        <div>
+          {!isDefault && (
+            <NavigateButton type="button" onClick={handleNavigateClick} />
+          )}
+          <h3>{title}</h3>
+          {tooltip && <Tooltip tip={tooltip} />}
+        </div>
+        {address && (
+          <Address
```

Comment:
Aren't `target` and `rel` already part of that component? They should be either way, sensible defaults

Reply:
This is a different component (styled a link) to the Address component defined elsewhere in the app

Reply:
Ah ok :+1: